SS_popdyn.tpl

```diff
@@ -1226,11 +1232,10 @@ FUNCTION void get_time_series()
               } // close fishery
             } //  end continuous F method
           }
-          else
-          // F_Method is Pope's approximation
+          else  //  doing F with Pope's approximation.  Predators cannot be used
           {
             //  SS_Label_Info_24.3.3.1 #Use F_Method=1 for Pope's approximation
-            //  SS_Label_Info_24.3.3.1.1 #loop over fleets
+            //  SS_Label_Info_24.3.3.1.1 #note that pred_M2 not implemented for Pope's
```

Comment:
There should be an error that prevents users trying to use Pope and pred_M2 together, not sure if there is already or not.